src/webportal/src/app/user/fabric/user-profile.jsx

```diff
@@ -210,6 +250,29 @@ const UserProfile = () => {
           >
             <TokenList tokens={tokens} onRevoke={onRevokeToken} />
           </UserProfileCard>
+          <UserProfileCard
```

Comment:
Please move this `UserProfileCard` before the `UserProfileCard` of Tokens.

SSH is supposed to be used more frequently than tokens.

Reply:
ok